Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/FormAMat/Index.cshtml

```diff
@@ -17,12 +17,12 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <h2 class="govuk-heading-l govuk-!-margin-bottom govuk-!-font-size-36">School application form</h2>
-            <nav class="gem-c-contents-list" aria-label="Pages in this guide" role="navigation" data-module="gem-track-click">
+		<nav class="gem-c-contents-list" aria-label="Pages in this guide" role="navigation" data-module="gem-track-click" data-cy="content-Table">
```

Comment:
We should probably have some consistency in the tags i.e. all lowercase and using either `-` or `_`. I'm not fussy which, but will make it easier to maintain

Reply:
will do :)